dev/NumberBox/NumberBox.xaml

```diff
@@ -79,24 +79,41 @@
                             </ResourceDictionary>
                         </Grid.Resources>
 
+                        <Grid.RowDefinitions>
+                            <RowDefinition Height="Auto" />
+                            <RowDefinition Height="Auto" />
+                            <RowDefinition Height="Auto" />
+                        </Grid.RowDefinitions>
+                        
                         <Grid.ColumnDefinitions>
                             <ColumnDefinition Width="*" />
                             <ColumnDefinition Width="Auto" />
                             <ColumnDefinition Width="Auto" />
                         </Grid.ColumnDefinitions>
 
+                        <ContentPresenter x:Name="HeaderContentPresenter"
+                            Grid.Row="0"
+                            Grid.Column="0"
```

Comment:
these are the default values.

Reply:
If you don't like it I can definitely remove them, but I personally like having them there even though it's just the default, I feel like it makes it explicit where everything is.

Reply:
I removed them after all :)